cds_ils/minters.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# CDS-ILS is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS-ILS minters."""
+
+from invenio_pidstore.models import PersistentIdentifier, PIDStatus
+
+from cds_ils.documents.api import DOCUMENT_LEGACY_PID_TYPE
+
+
+def legacy_recid_minter(data, uuid):
```

Comment:
sorry for not noticing earlier:
you should pass `legacy_recid` as an argument, not in data dict, it wouldn't depend on a specific dictionary